app/channels/application_cable/connection.rb

```diff
@@ -11,6 +11,8 @@ def connect
     def find_verified_user
       if verified_user = env["warden"].user
```

Comment:
```suggestion
      case env["warden"].user
      when verified_user
        verified_user
      when current_user
        current_user
      else
        reject_unauthorized_connection
      end
```
thử xem, mà với cả dùng = như này để chi đó :v nếu là so sánh thì phải dùng == chứ, mà gán như này thì thà dùng như sau còn hơn

`env["warden"].user.present? ? env["warden"].user : reject_unauthorized_connection`

Reply:
vang a